rotkehlchen/chain/bitcoin/utils.py

```diff
@@ -107,21 +188,91 @@ def pubkey_to_p2sh_p2wpkh_address(data: bytes) -> BTCAddress:
 
 def pubkey_to_bech32_address(data: bytes, witver: int) -> BTCAddress:
     """
-    Bitcoin pubkey to bech32 address
```

Comment:
I would add in the docstring what is `witver`

Reply:
Witver also known as the witness version is simply the version of the address.

BECH32 -> Witness version of `0`
BECH32M -> Witness version of `>=1`